packages/truss/src/generate.ts

```diff
@@ -113,6 +113,10 @@ class CssBuilder<T extends ${Properties}> {
       return this.newCss({ selector: Breakpoints[arg] });
     }
   }
+
+  get onHover() {
+    return this.newCss({ selector: "&>:hover" });
```

Comment:
I don't think we need `&>`. Could just be ":hover". I don't think it'll hurt anything, as it'll result in the same style rule, but pretty sure this is unnecessary.

Reply:
Oh yeah, totally right on `>`, duh.

Afaiu `&` is still necessary, as that is a placeholder for "the generated class name"...

...but actually you're right, even just doing `:hover`, emotion will will output it as `emotion-0:hover { ... }`. Cool! Will updated!